src/libs/fileDownload/getImageResolution.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Returns image dimensions for picked File object
+ * File object is returned as a result of a user selecting files using the <input> element
```

Comment:
Please add more of an explanation here for the platform-specific usage of this file such as this being the best way to get the height/width of an image on web, and you can also explain why you used onload vs the other option you were considering (I remember you mentioned something about performance).

Reply:
updated.